saas/fsm/unsanctioned_spend/README.md

```diff
@@ -19,11 +19,21 @@ This policy has the following input parameters required when launching the polic
 - *Number of Days Back* - Any unsanctioned expenses discovered during this time period will raise an incident
 - *Email addresses to notify* - Email addresses of the recipients you wish to notify
 
-## Policy Actions
+## Prerequisites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```

Comment:
same as the others.  carried over from another policy

Reply:
updated